INGI2143 Exam Janvier (#777)
Gp2mv3/Syntheses: src/q7/cl-INGI2263/exam/2019/Septembre/All/cl-INGI2263-exam-2019-Septembre-All.tex

\documentclass[en]{../../../../../../eplexam}

> \hypertitle{}{7}{INGI}{2263}{2019}{Septembre}{All}

Reviewer: The title is missing.

Reviewer 2: ```suggestion
\hypertitle{Computational Linguistics}{7}{INGI}{2263}{2019}{Septembre}{All}
```

{Nicolas Vanvyve}

> {Pierre Dupont and Cedrik Fairon}

Reviewer: ```suggestion
{Pierre Dupont and Cédrick Fairon}
```

> \section{We are interested inthe BLEU metric used to assess machine translation systems.}

Reviewer: ```suggestion
\section{We are interested in the BLEU metric used to assess machine translation systems.}
```

\nosolution

\section{We are interested in the Brill POS tagger}
\subsection{Describe precisly what needs to be a prioro defined in the Brill tagger as oppased to what in automatically learned from a corpus}

\nosolution

\subsection{Discuss the benefits and limitations of the Btill POS tagger}

\nosolution

\section{The following senstences illustrate different types of restrictions on the paradigmatic and syntagmatic axis.}
\begin{enumerate}
	\item Action speak louder than qords.
	\item She'll be burning her bridges of she goes to work for their competitor.
	\item The term blue collar was first used in referance to trade jobs in 1924
\end{enumerate}

\subsection{For each sentence, explain the paradigmatic and syntagmatic restrictions that you can identify.}

\nosolution

\subsection{Explain why it is useful to take these restriction into account for building NLP applications.}

\nosolution

\subsection{In the first sentence, indentify all the smallest meaningful uits and name their type (being as presise as possible).}

\nosolution

\section{An algorithm receives string A as input and produces string B as output.}

> \begin{itemize}

Reviewer: Perhaps lex this output in order to make it pop out more.

> 	\item[A] "Python is very useful for processing textual data."

Reviewer 2: ```suggestion
	\item[A] ``Python is very useful for processing textual data.''
```

> 	\item[B] "Python is veri use for process textual data."

Reviewer 2: ```suggestion
	\item[B] ``Python is veri use for process textual data.''
```
